.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Report a bug.
+title: "[Bug] "
+labels: bug
+assignees: ''
+
+---
+
+\-------------------------------------------------------------------------------------------------
+**IS THIS BUG RELATED TO EXTERNAL TOOLS? (e.g. git, less, etc.)**
+
+If it is, there's a good chance we might need to know a little bit more about the environment
+which you're running `bat` on.
+
+**If you're on Windows:**  
+Please tell us about your:
+
+- Windows Version
```

Comment:
Maybe add "`bat` version" here as well? Also, can we please move the Windows section below the Linux/MacOS section? I believe it's still a rather small percentage of users.

Reply:
I moved it below, and I added the request for `bat --version` to the top above everything else. If people use the `info.sh` script, it will be duplicated in that too, but there shouldn't be any harm in doing so.